DEVELOPMENT.md

```diff
@@ -24,37 +21,6 @@ _Note:_ Running the `UI` and `IntegrationTests` schemes follows the same steps a
 
 ## Importing Header Files
 
-To maintain support for CocoaPods, Swift Package Manager, and Carthage, our Objective-C import statements need specific attention.
-
-While SPM, Carthage, and manual integrations use the same import style (`#import <BraintreeCore/BraintreeCore.h>`), CocoaPods requires a different syntax (`#import <Braintree/BraintreeCore.h>`). This is because CocoaPods creates a single Braintree framework out of the subspecs the merchant includes, whereas SPM, Carthage, and manual integrations treat each module as a separate framework, i.e., BraintreeCore, BraintreeCard, etc.
-
-Public headers for each module must live in the directory `Public/<MODULE_NAME>`. This allows SPM to use the same import syntax as Carthage (e.g., `<BraintreeCore/BraintreCore.h>`).
-
-We use if-else preprocessor directives to satisfy each dependency manager. See the below example for importing a **public header file**.
-
-```objc
-#if __has_include(<Braintree/BraintreeCore.h>) // CocoaPods
-#import <Braintree/BraintreeCore.h>
-#else // SPM or Carthage
-#import <BraintreeCore/BraintreeCore.h>
-#endif
-```
-
-### Importing internal headers
-
-In general, we avoid importing **internal headers from other modules**, but occasionally it's necessary. See the below example for importing an internal header file from another module:
-```objc
-#if __has_include(<Braintree/BraintreeAmericanExpress.h>) // CocoaPods
-#import <Braintree/BTAPIClient_Internal.h>
-
-#elif SWIFT_PACKAGE // SPM
-#import "../BraintreeCore/BTAPIClient_Internal.h"
-
-#else // Carthage
-#import <BraintreeCore/BTAPIClient_Internal.h>
-#endif
-```
-
 ### Importing a Swift module into Obj-C
```

Comment:
Leaving this section for now since it may be a helpful reference for us as we migrate the Drop-in to Swift. Alternatively, we can move this to our internal knowledge repo and remove it from here entirely.